app/models/patient.rb

```diff
@@ -822,4 +825,28 @@ def address_timezone_offset
       timezone_for_state('massachusetts')
     end
   end
+
+  # Use the cached attribute if it exists, if not query with count for performance
+  # instead of loading all dependents.
+  def head_of_household?
+    return head_of_household unless head_of_household.nil?
+
+    dependents_exclude_self.size.positive?
+  end
+
+  # After remove callback passes a parameter which is the object that was just removed, we don't need it
+  # so we just throw it away
+  def inform_responder(*)
+    initial_responder = responder_id_was
+    # Yield to save
+    yield
+    Patient.find(initial_responder).refresh_head_of_household unless initial_responder.nil? || initial_responder == responder.id
```

Comment:
I'm seeing a lot of the tests fail because of this line right now because the `responder` is nil.